### PR TITLE
[issues] filter_bar, switch: update styling to filter buttons fixes #4608

### LINKED ISSUE #4608: #6460 problem with selecting filters in project overview
**URL:** https://meinberlin-dev.liqd.net/projekte/
**user:** any
**expected behaviour:** To select a “kind of participation” or “project status” filter, I can click anywhere on the respective button.
**behaviour:** Filter only gets selected, when the text inside the button is clicked.
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?
Bildschirmfoto 2022-10-24 um 15 23 19 (2)

### PR DESCRIPTION
**Pull request by phillimorland**
accessibility clean up and more fixes from old library swap

### CODE REVIEW

**Review by fuzzylogic2000 — APPROVED** [review]
Nice fixes and cleanup! 
Just maybe a bit more code to be deleted?

File: meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html [hunk 1]
```diff
@@ -523,13 +523,13 @@ <h2 id="cl-buttons">Buttons</h2>
             <div class="cl-example">
                 <p>
                     <button class="btn btn--light" type="button">Light Button</button>
-                    <button class="btn btn--transparent" type="button">Transparent Button</button>
+                    <button class="btn btn--light" type="button">Transparent Button</button>
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html:526]: @fuzzylogic2000 actually I just found one place where we still use it (project overview 'More filters') but it is not accessible so I need to have a think there and maybe talk to Marcelo

[the remark this answers is not in the document]
Comment (phillimorland) [reply meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html:526]: will replace it for now and see what he says when show him dev :p

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**